plugins/main/public/react-services/check-index.ts

```diff
@@ -0,0 +1,10 @@
+import { SavedObject } from '.';
+
+export async function checkExistenceIndices(indexPatternId: string) {
```

Comment:
The "check" prefix can be ambiguous; it may be clearer to use the "verify" prefix instead, such as `verifyExistenceIndices`.

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/7001/commits/7e10153c2e0817a9ffbb5796031906781d7d6779